testsuite/integration/smoke/pom.xml

```diff
@@ -640,48 +585,84 @@
-                        <artifactId>maven-resources-plugin</artifactId>
-                        <version>${version.resources.plugin}</version>
-                        <executions combine.self="override">
-                            <execution>
-                                <id>ts.copy-wildfly</id>
-                                <phase>generate-test-resources</phase>
+                                <phase>generate-resources</phase>
+                                <configuration>
+                                    <provisioning-dir>${project.build.directory}/wildfly</provisioning-dir>
+                                    <feature-packs>
+                                        <feature-pack>
+                                            <groupId>${testsuite.ee.galleon.pack.groupId}</groupId>
+                                            <artifactId>${testsuite.ee.galleon.pack.artifactId}</artifactId>
+                                            <version>${testsuite.ee.galleon.pack.version}</version>
+                                            <inherit-configs>false</inherit-configs>
+                                            <inherit-packages>false</inherit-packages>
+                                            <included-configs>
+                                                <config>
+                                                    <model>standalone</model>
+                                                    <name>standalone.xml</name>
+                                                </config>
+                                                <config>
+                                                    <model>standalone</model>
+                                                    <name>standalone-full.xml</name>
+                                                </config>
+                                                <config>
+                                                    <model>standalone</model>
+                                                    <name>standalone-microprofile.xml</name>
+                                                </config>
+                                            </included-configs>
+                                        </feature-pack>
+                                    </feature-packs>
+                                </configuration>
                             </execution>
                         </executions>
                     </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>standalone.microprofile.profile</id>
+            <activation>
+                <property>
+                    <name>ts.standalone.microprofile</name>
+                </property>
+            </activation>
+            <properties>
+                <!-- We don't want to use the default jboss.dist for this module, which is provisioned locally -->
+                <jboss.dist>${jbossas.project.dir}/${wildfly.build.output.dir}</jboss.dist>
```

Comment:
I think this now should be the default setting, i.e. what's in L33 of the pom. Then server-provisioning-profile switches it to what's currently in L33. This standalone.microprofile.profile doesn't need to deal with it.